docs/getting-started.md

```diff
@@ -38,6 +38,8 @@ These instructions assume you have a basic understanding of Linux and the comman
 
 We currently only support Linux, specifically Ubuntu 18.04 LTS. You are welcome to use a different version or distribution of Linux, but may need to make some tweaks in order for things to work.
 
+You can use Ubuntu 18.04 LTS inside Windows through Windows Subsystem for Linux, by following [this guide](./getting-started-wsl.md).
```

Comment:
I think that we should say that this is experimental and might require some effort to get working

Reply:
updated with some disclaimers